source/test/default_mode_test/SymWorld.test.cc

```diff
@@ -543,122 +442,87 @@ TEST_CASE( "Update", "[default]" ){
+        int res_change = res_after_update - res_before_update;
+        REQUIRE(res_per_update == res_change);
       }
     }
 
 
     WHEN("free living syms are allowed"){
-      int resPerUpdate = 80;
-      config.RES_DISTRIBUTE(resPerUpdate);
-      config.FREE_SYM_RES_DISTRIBUTE(resPerUpdate);
-      w.Resize(4,4);
+      res_per_update = 80;
+      config.RES_DISTRIBUTE(res_per_update);
+      config.FREE_SYM_RES_DISTRIBUTE(res_per_update);
+      world_size = 16;
+      w.Resize(world_size);
       w.SetFreeLivingSyms(1);
       config.FREE_LIVING_SYMS(1);
       config.MOVE_FREE_SYMS(1);
 
       WHEN("there are no syms in the world"){
         THEN("hosts process normally"){
-          host = new Host(&random, &w, &config, int_val);
           w.AddOrgAt(host, 0);
           int orig_points = host->GetPoints();
           w.Update();
 
-          REQUIRE(host->GetPoints() - orig_points == resPerUpdate);
+          REQUIRE(host->GetPoints() - orig_points == res_per_update);
         }
       }
 
-      WHEN("lysis is permitted, and thus phage are used"){
-        LysisWorld lw(random);
-        lw.Resize(4,4);
-        lw.SetFreeLivingSyms(1);
-
-        config.LYSIS(1);
-        config.LYSIS_CHANCE(1);
-        int burst_time = 2;
-        config.BURST_TIME(burst_time);
-        emp::Ptr<Organism> p = new Phage(&random, &lw, &config, int_val);
+      config.LYSIS(0);
+      config.HORIZ_TRANS(1);
+      world_size = 9;
+      w.Resize(world_size);
 
-        WHEN("there are no hosts"){
-          THEN("phage don't reproduce or get points on update"){
-            lw.AddOrgAt(p, 0);
-
-            int orig_num_orgs = lw.GetNumOrgs();
-            int orig_points = p->GetPoints();
-
-            for(int i = 0; i < 4; i ++){
-              lw.Update();
-            }
-
-            int new_num_orgs = lw.GetNumOrgs();
-            int new_points = p->GetPoints();
+      THEN("if only syms in the world they can get resources and reproduce"){
+        emp::Ptr<Organism> sym = emp::NewPtr<Symbiont>(&random, &w, &config, int_val);
+        w.SymDoBirth(sym, 0);
+        REQUIRE(w.GetNumOrgs() == 1);
 
-            REQUIRE(new_num_orgs == orig_num_orgs);
-            REQUIRE(new_points == orig_points);
-          }
+        for(int i = 0; i < num_updates; i++){
+          w.Update();
         }
+        //the sym has reproduced at least once
+        REQUIRE(w.GetNumOrgs() > 1);
 
-        WHEN("there are hosts"){
-          THEN("phage and hosts mingle in the world"){
-            lw.AddOrgAt(host, 0);
-            lw.AddOrgAt(p, emp::WorldPosition(0,1));
-
-            for(int i = 0; i < 5; i++){
-              lw.Update();
-            }
-
-            REQUIRE(lw.GetNumOrgs() == 2);
+        int num_pop_elements = 0;
+        for(int i = 0; i < world_size; i++){
+          if(w.GetPop()[i]){
+            num_pop_elements++;
           }
         }
-      }
+        REQUIRE(num_pop_elements == 0);
```

Comment:
Unclear what this is actually checking for, just that there aren't any 'hosts' in the host population?